lib/import_school_group_data.rb

```diff
@@ -0,0 +1,69 @@
+require 'csv'
+require 'httparty'
+
+SCHOOL_GROUP_URL = 'https://ea-edubase-api-prod.azurewebsites.net/edubase/downloads/public/allgroupsdata.csv'
```

Comment:
do these URLs need to be env vars to point at diff environments fo things like integration testing? (cause we havent got enough environment variables)

Reply:
We want the full list of data on all environments. Why would you want a different source of data for staging?

Reply:
i dont why you would tbh, just posing the question 👍

Reply:
i guess i was thinking along the line of if the data source was versioned. you might want to test against a new version, was the sort of thing i was thinking, but if thats not the case then no bother